src/lib/y2packager/dialogs/addon_selector.rb

```diff
@@ -264,7 +264,7 @@ def preselected_upgrade_products
       # @return [Array<Y2Packager::ProductLocation>] the products
       def preselected_installation_products
         default_modules = Yast::ProductFeatures.GetFeature("software", "default_modules")
-        return [] unless default_modules
+        return [] if default_modules == ""
```

Comment:
Maybe we should check also for `nil` or possibly other types with:

```suggestion
        return [] unless default_modules.is_a?(Array)
```